crypto/CMakeLists.txt

```diff
@@ -924,3 +942,13 @@ install(EXPORT crypto-targets
     DESTINATION "${CMAKE_INSTALL_LIBDIR}/crypto/cmake/${TARGET_DIR}"
     NAMESPACE AWS::
     COMPONENT Development)
+
+if(PERFORM_SONAME_BUILD)
+install(CODE "
+set(TGT lib${CRYPTO_LIB_NAME}.so.${ABI_VERSION})
+set(LNK \$ENV{DESTDIR}${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/libcrypto.so)
+message(STATUS \"Creating symlink: \${LNK} → \${TGT}\")
+execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink \${TGT} \${LNK})
+"
+  COMPONENT Development)
```

Comment:
Minor nit: Indentation

Reply:
Though for the ssl version of this I clearly goofed and was inconsistent with it :)